src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -358,11 +360,11 @@ Now we prove this definition is equivalent to the original one.
 ## Segal function and extension types
 
 Using the new characterization of Segal types, we can show that the type of
-functions or extensions into a family of Segal types is again a Segal type.
+functions or extensions into a family of Segal types is again a Segal type. For
+instance if X is a type and A : X -> U is such that A(x) is a Segal type for all
+x then (x : X) -> A x is a Segal type.
```

Comment:
You can format math using latex by writing 
````
instance if $X$ is a type and $A : X -> U$ is such that $A(x)$ is a Segal type for all
$x$ then $(x : X) -> A(x)$ is a Segal type.
````
Also swap `->` with the unicode alternative or `\to`

Reply:
Easy enough. Should that be our default style? It works for me.